scripts/import_cloudwatch_log_groups.py

```diff
@@ -0,0 +1,83 @@
+import json
+
+import boto3
+from more_itertools import (
+    one,
+)
+
+from azul import (
+    config,
+    logging,
+)
+from azul.logging import (
+    configure_script_logging,
+)
+from azul.terraform import (
+    terraform,
+)
+
+log = logging.getLogger(__name__)
+
+
+def main():
+    def add_prefix(name):
+        return 'aws_cloudwatch_log_group.' + name
+
+    log_groups = {}
+    tf_component = config.terraform_component
+
+    if tf_component == 'shared':
+        if config.slack_integration:
+            # Chatbot log groups
+            name = add_prefix('chatbot')
+            log_group = '/aws/chatbot/' + config.qualified_resource_name('chatbot')
+            log_groups[name] = log_group
```

Comment:
The presence of these comments indicates to me that this function is getting too big and ought to be split up. How about adding a new function for each type of log group and merging their return values in `main`, e.g.:

```suggestion
        def chatbot_log_groups() -> Iterable[tuple[str, str]]:
            if config.slack_integration:
                name = add_prefix('chatbot')
                log_group = '/aws/chatbot/' + config.qualified_resource_name('chatbot')
                yield name, log_group
        ...
        log_groups.update(chatbot_log_groups())
```

etc. for Lambda and API Gateway. I think it would make the entire script much easier to read.